pkg/loader/load_test.go

```diff
@@ -58,6 +59,7 @@ func (cs *LoadSuite) TestGetExecutor(c *check.C) {
 		metrics: &MetricsGroup{
 			QueryHistogramVec: &prometheus.HistogramVec{},
 		},
+		ctx: context.Background(),
```

Comment:
The `cancel` field needs to be non-null to avoid `s.cancel()` crashing, right?

Reply:
yes

Reply:
but the unit test dosen't call loader.Close(), no crashing now